test/fuzz/transformation_record_synonymous_constants_test.cpp

```diff
@@ -129,12 +129,14 @@ TEST(TransformationRecordSynonymousConstantsTest, IntConstants) {
   ApplyTransformationAndCheckFactManager(13, 22, context.get(),
                                          &transformation_context);
 
+  // TODO (https://github.com/KhronosGroup/SPIRV-Tools/issues/3536):
+  // Relax type check for integers.
```

Comment:
"TODO (" -> "TODO("

Reply:
Also say: "Uncomment this code once the issue is fixed."

Reply:
Done